packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/FormHead/index.js

```diff
@@ -69,6 +69,7 @@ const FormHead = ({
           })}
         </Link>
       }
+      ellipsis
```

Comment:
didn't we identify this isn't enough? so when you shrink the header even by a few pixels it pushes the actions off the screen?

Reply:
yes I have also a PR coming for the DS part

Reply:
I assume you're going to need that released before this PR can be merged then?

Reply:
yes you are right

Reply:
but I tested it locally with the new DS and it works, so for the QA phase we need to wait for the release of the DS, instead, for the review the code is ready